### ISSUE TITLE
This sample is out of date, uses Microsoft.Azure.Storage instead of newer Azure.Storage

### ISSUE THREAD

**Issue by patrickjlee** [Azure-Samples/storage-dotnet-sas-getting-started#5]
I like this sample, but it is out of date since it uses the Microsoft.Azure.Storage package which has been deprecated in favour of Azure.Storage.

Nuget Package manager says this about Microsoft.Azure.Storage.Blob:

> This library has been replaced by the following new Azure SDKs. You can read about the new Azure SDKs at https://aka.ms/azsdkvalueprop.
> 
>       The latest libraries to interact with the Azure Storage service are:
> 
>       * https://www.nuget.org/packages/Azure.Storage.Blobs
>       * https://www.nuget.org/packages/Azure.Storage.Queues/
>       * https://www.nuget.org/packages/Azure.Storage.Files.Shares/
> 
>       It is recommended that you move to the new package.
> 

I have tried using similar code to that in this sample with the new Azure.Storage libraries, but without any success, as the changes are far from obvious.  Please would you provide an updated version of this sample that works with the new packages?  
Thank you.

**Comment by graphicsxp** [comment]
I agree. Please provide a new sample with the new SDK packages. It is very confusing and difficult to figure out how to get a SAS from azure Storage.
